internal/project/handlers/mytest.go

```diff
@@ -1,10 +1,11 @@
-{{ .Copyright }}
+/**#bean*/ /*#bean.replace({{ .Copyright }})**/
 package handlers
 
 import (
 	"net/http"
 
-	"{{ .PkgPath }}/services"
+	/**#bean*/
+	"demo/services" /*#bean.replace("{{ .PkgPath }}/services")**/
```

Comment:
put these two lines in one line

Reply:
`go fmt` command make it, `go fmt` will run before I commit,but I don't know the fmt rules
bean macro still works in two lines